apps/opencs/model/world/columns.cpp

```diff
@@ -291,7 +291,7 @@ namespace CSMWorld
             { ColumnId_UChar, "Value [0..255]" },
             { ColumnId_NpcMisc, "NPC Misc" },
             { ColumnId_Level, "Level" },
-            { ColumnId_NpcFactionID, "Faction ID" },
+            // { ColumnId_NpcFactionID, "Faction ID" },
```

Comment:
Just remove the lines (here and in esmtool). It's not necessary to comment anything out.
Maybe rename mFactionId variable to something like "mUnknown" or "mPadding" and leave a comment saying "always 0".